CSSampleApp/src/main/java/com/example/androidsampleapp/MainActivity.kt

```diff
@@ -111,6 +111,10 @@ class MainActivity : AppCompatActivity() {
         Analytics.tagScreen("ActivityDialog_DialogShow")
     }
 
+    fun openCompose(view: View) {
```

Comment:
Unused parameter

Reply:
Yes, unfortunately. It's like for other existing onCLick handlers: they are defined in layout with android:onClick="openCompose" and then function implementing it needs to have this view parameter in order to be matched with that definition in layout. Although parameter is not used.
I made it in the same way as it was done for all other options on that screen, but it should probably all be refactored.